src/js/medium-editor.js

```diff
@@ -978,8 +1031,18 @@ else if (typeof define === 'function' && define.amd) {
                     this.toolbar.style.top = containerTop - this.toolbar.offsetHeight + "px";
                 }
 
-                this.toolbar.style.left = containerRect.left + "px";
-
+                if (this.options.toolbarAlign) {
```

Comment:
Perhaps you could factor out this `toolbarAlign` stuff into a separate pull request?  I would imagine it would be a lot easier to merge a pull request like that, then when it's attached to the form stuff you've added (which is very cool, but also significantly more complicated and impactful).  Just a thought though.